test/modules/corrado/import-ambiguity-bug/otherModule.future

```diff
@@ -0,0 +1,4 @@
+compiling a file adjacent to a module named "search.chpl" when importing the
+Time module causes an import ambiguity error and an internal compiler error
+
+See #23100
```

Comment:
This doesn't use our normal future formatting.  Typically we start with:
```
<category>: short description
#<issue number>

optional longer description
```
https://chapel-lang.org/docs/latest/developer/bestPractices/TestSystem.html#a-test-that-tracks-a-failure states this and test/functions/ferguson/hijacking/issue-18081.future has an actual example of it.